modules/custom/openy_activity_finder/js/programs_search.js

```diff
@@ -494,6 +618,42 @@
           component.availabilityPopup.price = component.table[index].price;
         });
       },
+
+      convertMonthesToYears: function(ages) {
```

Comment:
why on frontend? Could we fix this in backend ( we did this in YGTC ) It is one line in PHP
@jediandrey https://github.com/ymcatwincities/ymca/pull/3672/files#diff-1850d6d8e14f9cab18333e1915e6abe7R281 see here

Reply:
fixed (removed)